src/krkn_lib/tests/test_krkn_elastic.py

```diff
@@ -0,0 +1,21 @@
+import os
+import datetime
+from krkn_lib.telemetry.elastic import KrknElastic
+from krkn_lib.tests import BaseTest
+from krkn_lib.utils.safe_logger import SafeLogger
+
+class TestKrknElastic(BaseTest):
+    url = os.getenv("ES_SERVER")
+    safe_logger: SafeLogger = SafeLogger()
+
+    def test_upload_correct(self):
+        elastic = KrknElastic(self.safe_logger,self.url)
+        time = elastic.upload_data_to_elasticsearch( {"timestamp":datetime.datetime.now()},"chaos_test")
+        
```

Comment:
in this way we don't low down the coverage because the methods are referenced, but this are not actually testing functionalities. I understand that we don't (still) have an ELK deployed on the Action's KinD, do you promise that we try to improve them?

Reply:
I did add an actual reference to an elastic search server that is public so it is testing the functionality. That's why I added the secret here: https://github.com/krkn-chaos/krkn-lib/blob/67365d2e9b582f52902d9114a4927428aae46b8d/.github/workflows/build.yaml#L81

Reply:
probably should validate the response code though. Will work on adding this